ibis/backends/base/file/__init__.py

```diff
@@ -134,6 +134,25 @@ def connect(self, path):
     def version(self) -> str:
         return pd.__version__
 
+    def list_tables(self, path=None, like=None):
+        """
```

Comment:
if you override the doc-string then you need to make a full doc-string. also pls type all args

Reply:
Good point, updated.